### ISSUE TITLE
Story block: accessibility

### ISSUE THREAD

**Issue by bluefuton** [Automattic/jetpack#17224]
#### Steps to reproduce the issue

1. Embed the new beta WP Stories block into a page.
2. Visit the page with a screen reader or inspect with your browser's a11y tools.

#### What I expected

* A label or alternative text describing the story link.
* Expand button should be an interactive element like a button, not a div.

**Screenshots**
<img width="283" alt="93729782-0adfa680-fc1a-11ea-9ac4-e750824fc1dd" src="https://user-images.githubusercontent.com/17325/93834253-4ab49580-fccf-11ea-8ecc-95020cb5ddfe.png">

I believe @Tug is doing some work on this in https://github.com/Automattic/jetpack/pull/16852.

Discovered whilst testing stories in Reader for https://github.com/Automattic/wp-calypso/pull/45397.

cc @aforcier

**Comment by aforcier** [comment]
Thanks @bluefuton!

**Comment by aforcier** [comment]
For the expand button, since it's not the actually interactable element, let's try adding `role=img` to the SVG?

**Comment by Tug** [comment]
Should be fixed by https://github.com/Automattic/jetpack/pull/16852
Feel free to re-open if you still see the issue.
